code/datums/elements/falling_hazard.dm

```diff
@@ -0,0 +1,80 @@
+/// An element that will make a target thing do damage to any mob that it falls on from a z-level above
+/datum/element/falling_hazard
+	element_flags = ELEMENT_BESPOKE
```

Comment:
why use an element if it's just a component?

Reply:
They said this is because the falling protection was an element too which I guess is fineeeeeee